packages/EasyErrorHandler/src/Bridge/Symfony/Interfaces/TraceableErrorHandlerInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Interfaces;
+
+use EonX\EasyErrorHandler\Interfaces\ErrorHandlerInterface;
+use Throwable;
+
+interface TraceableErrorHandlerInterface extends ErrorHandlerInterface
+{
+    public function getReportedException(): ?Throwable;
```

Comment:
@ERuban applications could report more than one error per request, so this should return an array.

Also the class is called "ErrorHandlerInterface" so this method should say "getReportedErrors()".

And last one, what about the rendered errors? Applications could use the render method to render more than one errors without necessarily returning them to the consumers, and we might want to collect them too.

Reply:
Done